.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "e2e/submodules/poa-bridge-contracts"]
 	path = submodules/poa-bridge-contracts
-	url = https://github.com/poanetwork/poa-bridge-contracts.git
+	url = https://github.com/ColuLocalNetwork/poa-bridge-contracts.git
```

Comment:
After https://github.com/poanetwork/poa-bridge-contracts/pull/140 is merged we should update submodule

Reply:
off course :)